activerecord/test/cases/adapters/postgresql/range_test.rb

```diff
@@ -363,6 +380,16 @@ def test_ranges_correctly_escape_input
     end
   end
 
+  def test_ranges_correctly_unescape_output
+    @connection.execute(<<~SQL)
+      INSERT INTO postgresql_ranges (id, string_range)
+      VALUES (106, '["ca""t","do\\\\g")')
+    SQL
+
+    escaped_range = PostgresqlRange.find(106)
+    assert_equal('ca"t'...'do\\g', escaped_range.string_range)
```

Comment:
Is it possible to find the cause why the assertion is passed even if actually `string_range` is not inserted?

Reply:
Figured it out. This test suite's `setup` block silently `skip`s the test case if there is a `StatementInvalid` exception during creating the table and types. Because at that point I had not yet fixed the `teardown` to include `DROP TYPE IF EXISTS stringrange`, this exception handler was hit on subsequent tests with a `#PG::DuplicateObject: ERROR:  type "stringrange" already exists`, and so after the first test, the rest of the whole test suite was being skipped.

Fortunately I had already fixed this error, so the current branch actually runs the tests.